unlock-wordpress-plugin/inc/classes/fullpostpage/class-unlock-box-fullpp.php

```diff
@@ -0,0 +1,172 @@
+<?php
+/**
+ * Unlock box dynamic class for full post/page.
+ *
+ * @since 3.0.0
+ *
+ * @package unlock-protocol
+ */
+
+namespace Unlock_Protocol\Inc\FullPostPage;
+
+use Unlock_Protocol\Inc\Traits\Singleton;
+use Unlock_Protocol\Inc\Unlock;
+
+/**
+ * Class Unlock_Box_Fullpp
+ *
+ * @since 3.0.0
+ */
+class Unlock_Box_Fullpp {
+
+    use Singleton;
+
+    /**
+     * Construct method.
+     *
+     * @since 3.0.0
+     */
+    protected function __construct() {
+
+        $this->setup_hooks();
+
+    }
+
+    /**
+     * Setup hooks.
+     *
+     * @since 3.0.0
+     */
+    protected function setup_hooks() {
+		/**
+		 * Filters.
+		 */
+		add_filter( 'the_content', array( $this, 'trigger_unlockprotocol_flow' ) );	
+    }
+	
+	/**
```

Comment:
Formatting is strange here. Please use consistent spaces

Reply:
Not sure how that come to be may be old commit but the new one is well aligned. kindly recheck